services/src/datasets/external/gfbio.rs

```diff
@@ -40,35 +41,6 @@ use snafu::ensure;
 pub const GFBIO_PROVIDER_ID: DataProviderId =
     DataProviderId::from_u128(0x907f_9f5b_0304_4a0e_a5ef_28de_62d1_c0f9);
```

Comment:
- run the backend with the nfdi feature enabled
- run the frontend with my PR https://github.com/geo-engine/geoengine-ui/pull/541 and the gfbio dashboard
- use this link to access the test collection: http://localhost:4200/#/?collectionId=63cf68e4-6e11-469d-8f35-af83ee6586dc

Reply:
Ok, but do you also get errors here?

![image](https://user-images.githubusercontent.com/4947061/211203815-b53378a9-3707-4d2e-a356-9c237da5ffc0.png)

Reply:
Fixed the errors, needed to harvest all ABCD datasets.

For the ABCD dataset, it looks like we add the same thing twice. Is this intended or are these subsets?

Reply:
you are right. The entries are not ABCD datasets, but ABCD units.

The unit id is given in the `__id` field of the result, here e.g "urn:gfbio.org:abcd:3_259_402:**ZFMK+Sc0612**". (The final part of the id is missing if the whole dataset is meant.)

However, in the database the id is "ZFMK Sc0612" with a space instead of a plus.

So I could add a new collection item AbcdUnit and add a columnRangeFilter to the workflow where I take the id and replace the "+" with a space.

Shall I do this? I don't know what we agreed to support. Maybe adding the whole collection is fine?

For the larger collections if would also be smarter to move the id filter into the query string. But that would require more work.